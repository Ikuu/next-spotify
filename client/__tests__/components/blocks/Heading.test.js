import React from 'react';
import { create } from 'react-test-renderer';
import Heading from '../../../components/blocks/Heading';

test('H1 Renders', () => {
  const tree = create(<Heading>Sample Text</Heading>).toJSON();
  expect(tree).toMatchSnapshot();
});

test('H2 Renders', () => {
  const tree = create(<Heading level={2}>Sample Text</Heading>).toJSON();
  expect(tree).toMatchSnapshot();
});
