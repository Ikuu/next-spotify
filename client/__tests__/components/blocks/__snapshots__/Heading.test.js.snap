// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H1 Renders 1`] = `
<h1>
  Sample Text
</h1>
`;

exports[`H2 Renders 1`] = `
<h2>
  Sample Text
</h2>
`;
